### DISCUSSION TITLE
warnings

### DISCUSSION THREAD

**Discussion by MarcRohrer** [joaotavora/sly#607] [General]
H,

I get 

emacs/.emacs.d/elpa/sly-20230523.1254/lib/sly-buttons.el: Warning: Use keywords rather than deprecated positional arguments to `define-minor-mode'
emacs/.emacs.d/elpa/sly-20230523.1254/lib/sly-completion.el: Warning: Use keywords rather than deprecated positional arguments to `define-minor-mode'
emacs/.emacs.d/elpa/sly-20230523.1254/lib/sly-completion.el: Warning: ‘with-displayed-buffer-window’ is an obsolete macro (as of 28.1); use ‘with-current-buffer-window’ with action alist entry ‘body-function’.
emacs/.emacs.d/elpa/sly-20230523.1254/sly.el: Warning: Use keywords rather than deprecated positional arguments to `define-minor-mode' [5 times]

starting Emacs 28.2 under Windows.
Maybe adapt as you go....

best wishes!

Marc

**Comment by cxwx** [comment]
same issue
